src/reducers/versions.tsx

```diff
@@ -669,13 +669,28 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     case getType(actions.updateSelectedPath): {
```

Comment:
It feels wrong that parent directories are only added to `expandedPaths` on `updateSelectedPath` and not when `version.selectedPath` is [first created](https://github.com/mozilla/addons-code-manager/blob/e91ed7269bc47cf15ecae73cb6cce44a1fbb803e/src/reducers/versions.tsx#L639). Adding `selectedPath` to `expandedPaths` at creation time would expand the tree when the page first loads which seems like the right thing to do. I don't understand why anyone would want to start the page with a collapsed tree.

This is not a change request, just me thinking out loud.

Reply:
I agree that we should do this as well, but I take from your final sentence that you feel this could be a follow-up?

Reply:
Yeah, no change request. I'm not really sure what to do about it yet. I wonder if storing the initial file at `version.defaultFile` would just make everything more clear.

Reply:
I thought your original thought made a lot of sense. I.e., when we first store `selectedPath` for a version, we also add all of its parents to `expandedPaths`. That seems pretty simple and straightforward.